menas-web/ui/theme/sap/tnt/themes/sap_belize/library-parameters.json

```diff
@@ -0,0 +1 @@
+{"default":{"_sap_tnt_ToolHeader_ITHOverflowButtonTop":"0.5rem","_sap_tnt_ToolHeader_Height":"3rem","_sap_tnt_ToolPage_AsideWidth":"15rem","_sap_tnt_ToolPage_AsideWidthOnTablet":"3rem","_sap_tnt_ToolPage_HeaderHeight":"3rem","_sap_tnt_ToolPage_FooterHeight":"3rem","_sap_tnt_ToolPage_CompactAsideWidth":"15rem","_sap_tnt_ToolPage_CompactAsideWidthOnTablet":"3rem","_sap_tnt_ToolPage_CompactHeaderHeight":"3rem","_sap_tnt_ToolPage_CompactFooterHeight":"2rem","_sap_tnt_ToolPage_AnimationDuration":"0.3s","_sap_tnt_InfoLabel_Margin":"0.9375rem","_sap_tnt_InfoLabel_NoTextPadding":"0.5625rem","_sap_tnt_InfoLabel_NoTextDisplayOnlyPadding":"0.5rem","_sap_tnt_InfoLabel_LineHeight":"1.125rem","_sap_tnt_InfoLabel_LineHeightDisplayOnly":"1rem","_sap_tnt_InfoLabel_BackgroundColor1":"#ffb200","_sap_tnt_InfoLabel_BackgroundColor2":"#fe995b","_sap_tnt_InfoLabel_BackgroundColor3":"#c14646","_sap_tnt_InfoLabel_BackgroundColor4":"#ab218e","_sap_tnt_InfoLabel_BackgroundColor5":"#4974bc","_sap_tnt_InfoLabel_BackgroundColor6":"#007db2","_sap_tnt_InfoLabel_BackgroundColor7":"#168282","_sap_tnt_InfoLabel_BackgroundColor8":"#68831d","_sap_tnt_InfoLabel_BackgroundColor9":"#666666","_sap_tnt_InfoLabel_ColorsArray":"#ffb200, #fe995b, #c14646, #ab218e, #4974bc, #007db2, #168282, #68831d, #666666","value":"#666666"},"scopes":{"sapContrast":{"_sap_tnt_InfoLabel_BackgroundColor1":"#ffb914","_sap_tnt_InfoLabel_BackgroundColor2":"#ff995a","_sap_tnt_InfoLabel_BackgroundColor3":"#ff7878","_sap_tnt_InfoLabel_BackgroundColor4":"#f283da","_sap_tnt_InfoLabel_BackgroundColor5":"#93b5ed","_sap_tnt_InfoLabel_BackgroundColor6":"#19baff","_sap_tnt_InfoLabel_BackgroundColor7":"#23cccc","_sap_tnt_InfoLabel_BackgroundColor8":"#a9d432","_sap_tnt_InfoLabel_BackgroundColor9":"#bbbaba","_sap_tnt_InfoLabel_ColorsArray":"#ffb914, #ff995a, #ff7878, #f283da, #93b5ed, #19baff, #23cccc, #a9d432, #bbbaba","value":"#bbbaba"}}}
```

Comment:
where does this come from? Could it be formatted better?

Reply:
So the story is that I could not point to the npm dependency for the theme: `"@openui5/themelib_sap_belize": "^1.67.1"`because it lacks the library.css and library-parameters.json files which are build somehow but I could not figure out how, so I added the needed files to the source code. This can be considered a first step, but it's an open discussion how to properly provide these files.